app/config/drupal-demo/install.sh

```diff
@@ -116,7 +116,7 @@ EOPERM
     add 'register to volunteer'
 EOPERM
 
-  cv en civirules civisualize cividiscount
+  cv en civirules civisualize cividiscount org.civicrm.search
```

Comment:
we may want to wrap this in an if statement cos not all versions of civicrm will have the search kit code available

Reply:
what about an approach like this https://github.com/civicrm/civicrm-buildkit/pull/569/files#diff-f69ba7259c51f34bf3e1b9047bf97ca8R121

Reply:
* Yeah, doing some kind of batch selector is interesting.

    * There used to be support for selecting path (eg `cv api extension.install path=$PWD/ext/'*'`), but that's regressed.

    * Since we have 'tags' now, it might be better to [have a tag selector](https://github.com/civicrm/cv/pull/82), e.g.

        ```xml
        <tags>
          <tag>demo</tag>
        </tags>
        ```

        ```
        cv en @demo
        ```

* Wasn't the extension being renamed to `searchkit`?

* `cv en` has an option `--ignore-missing`. This may be actually be the simplest thing.

Reply:
+1 to "--ignore-missing"

Keep it simple :-)

Reply:
ohh ignore missing = yes!